webui/react/src/components/kit/Icon.tsx

```diff
@@ -140,12 +141,25 @@ const OptionsIcon: React.FC = () => (
 );
 
 const PanelIcon: React.FC = () => (
-  <svg fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+  <svg fill="none" height="20" viewBox="0 0 20 20" width="20" xmlns="http://www.w3.org/2000/svg">
+    <rect height="15.6667" rx="1.5" stroke="currentcolor" width="15.6667" x="2.16675" y="2.16663" />
     <path
-      clipRule="evenodd"
-      d="M10.72 5H19V19H10.72V5ZM9.72 5H5V19H9.72V5ZM10.72 4H19C19.5523 4 20 4.44772 20 5V19C20 19.5523 19.5523 20 19 20H10.72H9.72H5C4.93096 20 4.86356 19.993 4.79847 19.9797C4.34278 19.8864 4 19.4832 4 19V5C4 4.44772 4.44772 4 5 4H9.72H10.72Z"
+      d="M3.66675 2.16663H8.16675V17.8333H3.66675C2.83832 17.8333 2.16675 17.1617 2.16675 16.3333V3.66663C2.16675 2.8382 2.83832 2.16663 3.66675 2.16663Z"
+      stroke="currentcolor"
+    />
+  </svg>
+);
```

Comment:
Previous panel icon was changed because the newly supplied `panel-on` icon is of different size with `panel` icon.